views/forms/UserProfileForm/UserProfileForm.test.jsx

```diff
@@ -34,14 +32,16 @@ describe('User Profile Form', () => {
   test('calls the onSubmit prop when the form is submitted with valid data', async () => {
     const user = userEvent.setup()
     const onSubmit = jest.fn()
-    const props = { ...defaultProps, onSubmit }
+    const props = {
+      ...defaultProps,
+      onSubmit,
+      errors: { mail: { message: 'mail must be a valid email' } },
```

Comment:
why are we adding errors for the valid data test? Should the form submit with errors?

Reply:
[0730f7e](https://github.com/AMP-SCZ/dpdash/pull/478/commits/0730f7e99359b47d86b078fb3e0ad15f74bb8fcb)